mappings/net/minecraft/world/World.mapping

```diff
@@ -101,13 +109,31 @@ CLASS net/minecraft/class_101 net/minecraft/world/World
 		ARG 7 z2
 	METHOD method_229 spawnEntity (Lnet/minecraft/class_140;)V
 		ARG 1 entity
+	METHOD method_230 createExplosion (Lnet/minecraft/class_140;DDDF)V
+		ARG 1 entity
+		ARG 2 x
+		ARG 4 y
+		ARG 6 z
+		ARG 8 size
 	METHOD method_231 playSound (Lnet/minecraft/class_140;Ljava/lang/String;FF)V
 		ARG 1 entity
 		ARG 2 name
 		ARG 3 volume
 		ARG 4 pitch
+	METHOD method_232 getEntitiesinBox (Lnet/minecraft/class_140;Lnet/minecraft/class_156;)Ljava/util/List;
+		ARG 1 entity
+		ARG 2 box
+	METHOD method_233 getTargetFromVec3d (Lnet/minecraft/class_155;Lnet/minecraft/class_155;)Lnet/minecraft/class_157;
+		ARG 1 pos1
+		ARG 2 pos2
+	METHOD method_234 getTargetFromVec3d (Lnet/minecraft/class_155;Lnet/minecraft/class_155;Z)Lnet/minecraft/class_157;
+		ARG 1 pos1
+		ARG 2 pos2
+	METHOD method_236 getBlocksInBox (Lnet/minecraft/class_156;)Ljava/util/List;
+		ARG 1 box
 	METHOD method_240 recursivelyDelete ([Ljava/io/File;)V
 		ARG 0 filesInFolder
+	METHOD method_241 shouldBeIgnited ()Z
```

Comment:
I just realized: would it make more or less sense if this would be `shouldIgnite`?